src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.cs

```diff
@@ -730,6 +730,12 @@ private void UpdateInlines(string text)
 				e.Handled = true;
 				// hyperlink.CompleteGesture(); No needs to complete the gesture as the TextBlock won't even receive the Pressed.
 			}
+			else if (sender is TextBlock textBlock && textBlock.IsTextSelectionEnabled)
+			{
+				// Selectable TextBlock should also handle pointer pressed to ensure
+				// RootVisual does not steal its focus.
+				e.Handled = true;
```

Comment:
Is this the UWP behavior? I would suspect that would break some interaction in the app like a simple `Button.Click` ... even though there are usually not so much `TextBlock` flagged as `IsTextSelectionEnabled` so it's probably not a big deal.

Reply:
Yes, based on my UWP debugging, when the `TextBlock` has `IsTextSelectionEnabled` set to `true`, it does handle `PointerPressed`. Not happening when the property is `false`.